datalad/config.py

```diff
@@ -265,6 +294,7 @@ class ConfigManager(object):
       not global or system configuration are considered; if 'any'
       all possible sources of configuration are considered.
     """
+    # TODO: above dataset* -> branch*?
```

Comment:
and `dataset-local` -> `branch-local`

Reply:
ok, let's do those as well then.  And thank you for the timing results!